db/models.py

```diff
@@ -40,13 +53,78 @@
 
 
 class MovieSession(models.Model):
-    show_time = models.DateTimeField()
-    cinema_hall = models.ForeignKey(
+    show_time: models.DateTimeField = models.DateTimeField()
+    cinema_hall: models.ForeignKey = models.ForeignKey(
         to=CinemaHall, on_delete=models.CASCADE, related_name="movie_sessions"
     )
-    movie = models.ForeignKey(
+    movie: models.ForeignKey = models.ForeignKey(
         to=Movie, on_delete=models.CASCADE, related_name="movie_sessions"
     )
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at: models.DateTimeField = models.DateTimeField(auto_now_add=True)
```

Comment:
Type annotations for Django model fields are not necessary and could lead to confusion. It's better to use Django's field types directly without type annotations.